tests/launchChecklist.spec.js

```
@@ -0,0 +1,531 @@
+    }
+
+    // Sort opportunities by priority (critical > high > medium > low)
+    const priorityOrder = { critical: 0, high: 1, medium: 2, low: 3 }
+    opportunities.sort(
+      (a, b) =>
+        priorityOrder[a.priority || 'low'] - priorityOrder[b.priority || 'low']
+    )
+
+    return opportunities
+  },
+}
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)